src/app/wizard/step/cluster/component.ts

```diff
@@ -129,6 +133,7 @@ export class ClusterStepComponent extends StepBase implements OnInit, ControlVal
       [Controls.ProxyMode]: this._builder.control(''),
       [Controls.PodsCIDR]: new FormControl('', [CIDR_PATTERN_VALIDATOR]),
       [Controls.ServicesCIDR]: new FormControl('', [CIDR_PATTERN_VALIDATOR]),
+      [Controls.CNIPlugin]: new FormControl(''),
```

Comment:
Should it be required value?

Reply:
doesn't need to be, we autoconfigure on backend if it's empty